tc/graphene/python_worker/README.md

```diff
@@ -0,0 +1,114 @@
+<!--
```

Comment:
Th is existing file shown as new. Please fix

Reply:
Done